databricks_cli/jobs/api.py

```diff
@@ -39,6 +39,11 @@ def delete_job(self, job_id):
     def get_job(self, job_id):
         return self.client.get_job(job_id)
 
+    def get_jobs_by_name(self, name):
```

Comment:
Could you rename it to "_list_jobs_by_name" and move it to the bottom? This doesn't match any API endpoint.

Reply:
Since I'm planning to call this function from the stack api, isn't it bad practice to call functions from other classes that have a `_` in front? please advise.